<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryBuilderSqlWidget</class>
 <widget class="QWidget" name="QueryBuilderSqlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query builder SQL visualization</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="schem_qualif_tb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Schema-qualified</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Code display:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="code_options_cmb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>Aerated</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Condensed</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="manage_tb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Manage</string>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/managedb.png</normaloff>:/icones/icones/managedb.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="save_sql_tb">
       <property name="toolTip">
        <string>Save the SQL code to a file.</string>
       </property>
       <property name="text">
        <string>Save SQL</string>
       </property>
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icones/icones/salvar.png</normaloff>:/icones/icones/salvar.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="sqlcode_wgt" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
